app/__init__.py

```diff
@@ -60,8 +60,11 @@ def refresh_session():
 @login_manager.user_loader
 def load_user(user_id):
     user_json = data_api_client.get_user(user_id=int(user_id))
+
     if user_json:
-        return User.from_json(user_json)
+        user = User.from_json(user_json)
+        if user.is_active() and not user.is_locked():
+            return user
```

Comment:
I've done a bit of investigating and we can just do `return User.from_json(user_json)` but it would require a couple of changes in digitalmarketplace-utils. Could we catch up this tomorrow?